extensions/adobe/experience/decisioning/experienceevent-proposition-interaction.example.2.json

```diff
@@ -9,21 +9,8 @@
           "xdm:id": "experience1"
         }
       ],
-      "xdm:decisionEventID": "TNT:activity1:experience1:1597208433",
-      "xdm:decisionEventDetails": {
-        "xdm:experienceID": "experience1"
-      },
-      "xdm:decisionScope": "__view__",
-      "xdm:decisionScopeDetails": {
-        "xdm:id": "__view__",
-        "xdm:decisionProvider": "Target Premium",
-        "xdm:placement": {
-          "xdm:id": "mbox1"
-        },
-        "xdm:activity": {
-          "xdm:id": "actitivty1"
-        }
-      }
```

Comment:
Remove the deprecated properties from the example and only leave the ones that got renamed